sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/common/cloud-provider.adoc

```diff
@@ -0,0 +1,54 @@
+
+include:description.adoc[]
+
+==== Data Theft
+
+The malicious users now have the ability to steal various types of data stored
+in the Cloud services, such as customer information, intellectual property,
+financial data, or other sensitive data.
+
+This vulnerability affects a number of AWS services, including:
```

Comment:
```suggestion
This vulnerability affects a number of services, including:
```

Since the rule will be for all kind of secrets, I guess it does not make sense to talk about AWS here specifically.